main.tf

```diff
@@ -98,7 +113,7 @@ resource "aws_transfer_ssh_key" "default" {
 resource "aws_eip" "sftp" {
   count = local.enabled && var.eip_enabled ? length(var.subnet_ids) : 0
 
-  vpc = local.is_vpc
+  domain = local.is_vpc ? "vpc" : null
```

Comment:
@romga I'm assuming this was due to an upgrade of the AWS provider? These types of changes are not typically backwards compatible. Can you look up the version of the AWS provider in which this was added, update our pinning of that provider in `versions.tf`, and then run `make init` and `make readme`? We'll need to account for this and it'll require this PR to be a breaking change. Your other option will be to downgrade your local provider version, so that you don't need this change to be made.